modules/tinymce/src/plugins/table/test/ts/module/test/TableModifiersTestUtils.ts

```diff
@@ -78,64 +78,80 @@ const assertStructureHasCustomStyle = (editor: Editor, rows: number, columns: nu
     '</table>');
 };
 
-const closeMenu = (editor) =>
+const closeMenu = (editor) => {
   TinyUiActions.keydown(editor, Keys.escape());
+  TinyUiActions.keydown(editor, Keys.escape());
+};
 
-const openMenu = (editor: Editor, title: string) =>
-  TinyUiActions.clickOnToolbar(editor, `button[title="${title}"]`);
+const clickOnButton = async (editor: Editor, title: string, toolbar: boolean) => {
```

Comment:
Please make this 2 separate functions instead of trying to overload it like this. It makes it hard to read/understand what's happening with just a bunch of booleans. It's also confusing, as click on button has nothing to do with menu items.

- [ ] Break this up into a `clickOnButton` and `clickOnMenuItem` or similar functions